src/query/functions/aggregation/base.go

```diff
@@ -72,9 +72,10 @@ func NewAggregationOp(
 
 // baseOp stores required properties for the baseOp
 type baseOp struct {
-	params NodeParams
-	opType string
-	aggFn  aggregationFn
+	params      NodeParams
+	opType      string
```

Comment:
confusing to have both opType and opTypeBytes. Maybe just have a OpType struct and hide it behind that ? Also, these functions are created once per query so i'm not sure what we are trying to optimize here ? This definitely makes it harder to understand.

Reply:
Fair, will standardize